Update agent-configuration-files.md (#7730)
DataDog/documentation: content/en/agent/guide/agent-configuration-files.md

### Checks configuration files for Agent 6

> An example for each Agent check configuration file can be found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` in order to enable the associated check. Note that the Agent loads  valid YAML file contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:

Reviewer: ```suggestion
An example for each Agent check configuration file is found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` to enable the associated check. Note that the Agent loads valid YAML files contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This allows complex configurations to be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
```

```text
/etc/datadog-agent/conf.d/http_check.d/
├── backend.yaml
└── frontend.yaml
```

> A special case are YAML files with the suffix `.default`.  These files will be loaded by the Agent automatically and help define the core set of checks that are always enabled.  To turn off one of these checks, simply change the suffix to `.disabled` or remove the file altogether.

Reviewer: ```suggestion
A special case are YAML files with the suffix `.default`. These files are loaded by the Agent automatically and help define the core set of checks that are always enabled. To turn off one of these checks, change the suffix to `.disabled` or remove the file altogether.
```

> 

Reviewer 2: The main point is that they're only loaded if we can't find a configuration for the check. So if the user created a disk.yaml the disk.yaml.default won't be loaded. User should not have to remove or renamed them (the goal was for them to be mostly ignored by users).

Author: Makes sense, added a line suggesting the user copy the `.default` file to make any changes.

> A special case are YAML files with the suffix `.default`. These files are loaded by the Agent by default and help define the core set of checks that are always enabled. To turn off one of these checks, change the suffix to `.disabled` or remove the file altogether.  To customize these checks, copy the `conf.yaml.default` file to `conf.yaml` and make any desired changes, and the `.default` file will be ignored.
> 

Reviewer 2: Users should not use `.default` as their base configuration as the `.example` contains all the doc strings. `.default` should just be ignored by users.

What do you think of:

```suggestion
A special case are YAML files with the suffix `.default`. These files are loaded by the Agent by default and help define the core set of checks that are always enabled (CPU, memory, uptime ...). They are ignored if any other configuration are found for that check, therefor you can safely ignore them. If you want to disable one of the default check simply remove that file. To configure these checks, `conf.yaml.example` should be use as a base.
```
